openedx/features/content_type_gating/tests/test_access.py

```diff
@@ -416,6 +433,28 @@ def test_access_course_team_users(self, role_factory):
             request_factory=self.factory,
         )
 
+    @ddt.data(
```

Comment:
is there a reason this test needs to be separate from the previous one?  
it seems like it is just testing the same thing for additional roles